unix_integration/src/constants.rs

```diff
@@ -13,3 +13,4 @@ pub const DEFAULT_HOME_ALIAS: Option<HomeAttr> = Some(HomeAttr::Spn);
 pub const DEFAULT_USE_ETC_SKEL: bool = false;
 pub const DEFAULT_UID_ATTR_MAP: UidAttr = UidAttr::Spn;
 pub const DEFAULT_GID_ATTR_MAP: UidAttr = UidAttr::Spn;
+pub const DEFAULT_SELINUX: bool = true;
```

Comment:
Probably should default to false for now, at least until we have a wider test base. @Firstyear ?

Reply:
I'm fine with leaving it disabled as the default, but users running SELinux will need to know to enable it during installation or there will be problems.

Reply:
Yep, that can and should be called out in the documentation.

Reply:
I think better would be to actually try and detect at run time / start up if selinux is present on the system. We can probably use the selinux::boot_mode / selinux::current_mode calls to work this out. That way selinux support is enabled automatically when we are on a system that uses selinux.

Reply:
Actually I think this is already the case given how the default config is constructed. So I think default to true here is okay since it will automatically disable when selinux is not present.